src/website/app/demos/Flex/bestPractices.js

```diff
@@ -72,17 +73,17 @@ align components to a columnar layout. Use [Grid](/components/grid) and
       description: `Don't use Flex if the content being laid out should _not_
 [respond to language direction](#rtl).`,
       example: (
-        <Flex css={{ height: 200 }}>
+        <Flex className={css({ height: 200 })}>
```

Comment:
Flex (which I wrote) has a `height` prop, so I have no idea why this (which I wrote) was using `css` here. 🤦‍♂️

Reply:
Fixed.